src/main/java/net/hydromatic/sqllogictest/SltTestFile.java

```diff
@@ -104,8 +104,9 @@ public boolean accept(List<String> skip, List<String> only) {
    * Create a test file from the file with the specified name.
    */
   public SltTestFile(String testFile) throws IOException {
-    File file = new File(testFile);
-    this.reader = new BufferedReader(new FileReader(file));
+    ClassLoader clLoader = Thread.currentThread().getContextClassLoader();
```

Comment:
The main problem with this approach is that it will only run tests that are part of the resources. One feature I'd like to implement is to synthesize small reproductions for failures, which would be generated as new *.test files, but would not be in the resources.

Reply:
True but let's discuss about this when we decide to implement this new feature.